datalad/core/local/status.py

```diff
@@ -495,3 +495,24 @@ def custom_result_summary_renderer(results):  # pragma: more cover
                for r in results):
             from datalad.ui import ui
             ui.message("nothing to save, working tree clean")
+
+        # ### Report on possibly cleanable temporary locations
+
+        # Infer the dataset we ran on from the results and get original
+        # recursive (+limit) options:
+        refds = set(r['refds'] for r in results)
+        assert len(refds) == 1
+        refds = Dataset(refds.pop())
+        recursive = kwargs.pop('recursive', False)
+        recursion_limit = kwargs.pop('recursion_limit', None)
+        clean_infos = refds.clean(info=True,
+                                  recursive=recursive,
+                                  recursion_limit=recursion_limit,
+                                  return_type='list')
+        if any(r['action'] == 'clean-info' and r['status'] == 'ok'
+               for r in clean_infos):
+            # If status reported on cleanable temp locations, add a hint to
+            # datalad-clean:
+            from datalad.ui import ui
+            ui.message("Note: You can remove temporary files anytime by using "
+                       "datalad's 'clean' command.")
```

Comment:
We should have a style guide, as each time I am unsure what tone to use. Passive, active, imperative?

What about simply `Temporary data can be removed with DataLad's 'clean' command`?

I personally think these messages should be as concise and interpretation independent as possible.

Reply:
Git does not seem to use 'hint:' or 'note:' markup:

```
Changes not staged for commit:

[...]

It took 13.57 seconds to enumerate untracked files. 'status -uno'
may speed it up, but you have to be careful not to forget to add
new files yourself (see 'git help status').
```

Reply:
@adswa used yet another (establish) way to give hints https://github.com/datalad/datalad/pull/5726#issuecomment-860619676

which should declare one as *the one*.

Reply:
> We should have a style guide, as each time I am unsure what tone to use. Passive, active, imperative?

Agree. I struggle everytime.

Reply:
> @adswa used yet another (established?) way to give hints #5726 (comment)

Not established until merged, I'd say ;-)
However, agree - like the result record enhancement much better. Will use that, too.

Reply:
I "established" it in #4338 I think, but simply copied Git ;-)

Reply:
Huh. I forgot that. :)